### DISCUSSION TITLE
Builds stuck in "Queued" state

### DISCUSSION THREAD

**Discussion by chrisdrackett** [vercel/vercel#4866] [Help] [answered]
for the last hour all our builds seem stuck in the "queued" state. Is this happening to anyone else? I assume this is not an issue with our code or account, but could be wrong.

**Comment by styfle** [comment, marked as answer]
Hi @chrisdrackett 

Concurrent Builds will be queued depending on your team's Billing Settings.

![Concurrent Builds](https://user-images.githubusercontent.com/229881/87725699-c6540c80-c78b-11ea-84ae-9c2cc1decbc7.png)


You can cancel builds if you have too many concurrent builds. ([Read blog post](https://vercel.com/blog/canceling-ongoing-deployments))

If you don't have any other concurrent builds, then there might be another issue. Send a support [Support Request](https://vercel.com/support/request) so we can look at that specific deployment for you and see what the problem is, thanks!
